### ISSUE TITLE
First cell in a table cannot be tapped with VoiceOver

### ISSUE THREAD

**Issue by defagos** [SRGSSR/playsrg-ios#138]
When using VoiceOver, the first row in a table cannot be tapped (selection, but also long press). This issue was kindly reported by an SRF user:

> Der Fehler erscheint, wenn man auf dem iPhone die Sprachausgabe VoiceOver eingeschaltet hat. Wenn man nun zum Menüpunkt „Profil“ geht und dort „Favoriten“ anwählt, ist es nicht möglich den obersten Eintrag anzuwählen. Die weiteren Einträge funktionieren. Wenn man einen anderen Eintrag anwählt und dann wieder zurück geht, klappts auch mit dem obersten.
Gerne sende ich Ihnen in der Beilage einen kurzen Bildschirmvideo der das demonstriert.

### Issue type

Incorrect behavior

### Environment information

* Application version: 3.0.0 (but issue existed at least since 2.9.7)
* iOS version: 13 (also tested with iOS 12)
* Device: Physical

### Reproducibility

Always reproducible

### Steps to reproduce

1. Add a few shows to the favorites.
1. Go to the favorite list.
1. Enable VoiceOver
1. Move to the first cell. Attempt to tap it (does not work).
1. Move to the second cell. Attempt to tap it (works).

The issue also affects other lists (e.g. Downloads).

**Comment by defagos** [comment]
The issue is due to the refresh control stealing the focus. I disabled user interaction for it (it still works as expected) and ensured consistent z position with a small custom Play class.

**Comment by defagos** [comment]
Fix available for review on the `feature/voice-over-cell-fix` branch.

**Comment by defagos** [comment]
If I remove the table view insets and it works if the table is entirely at the top, without margin before the first cell. I guess that adding insets reveals the refresh control, giving it a chance to steal the focus.

**Comment by pyby** [comment]
First item can be open. Test passed on:
- iOS 9.3.4
- iOS 10.3.3
- iOS 12.4.3
- iOS 13.4
